DataProcessing/datax-host/pom.xml

```diff
@@ -107,7 +107,7 @@ SOFTWARE
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_2.11</artifactId>
-            <version>2.2.6</version>
+            <version>3.0.8</version>
```

Comment:
do we need to rev the versions of all componets to 1.3?

Reply:
@rohit489

Reply:
We should upgrade the version of all the packages to 1.3.0-SNAPSHOT